src/components/Layout/LayoutBox.js

```diff
@@ -0,0 +1,61 @@
+import React, { useMemo } from 'react'
+import PropTypes from 'prop-types'
+import { Box, useLayout, useTheme, GU } from '@aragon/ui'
+
+function LayoutBox({ children, heading, primary, mode, ...props }) {
+  const { layoutName } = useLayout()
+  const theme = useTheme()
+  const compactMode = layoutName === 'small'
+
+  const { backgroundColor, borderColor } = useMemo(() => {
+    const attributes = {
+      warning: {
+        backgroundColor: '#fefdfb',
+        borderColor: theme.warning,
+      },
+      negative: {
+        backgroundColor: '#FFFAFA',
+        borderColor: '#FF7C7C',
+      },
+      disabled: {
+        backgroundColor: theme.surfacePressed,
+        borderColor: theme.border,
+      },
+    }
```

Comment:
I'm using semantic naming here to keep this component generalised, the disputable states can then be easily mapped to use them.